docs/usage.rst

```diff
@@ -59,6 +59,46 @@ In case you want to explicitly pass the credentials from Python, use :py:class:`
     :members:
     :undoc-members:
 
+The ``Table`` class
+-------------------
+
```

Comment:
please add a note that the `Table` class should not be instantiated directly, but always via Client.table.